base/doc/ltnews41.tex

```diff
@@ -1060,12 +1160,15 @@ \subsection{\pkg{array}:\ Fix handling of empty p-cells}
 \githubissue{1730}
 
 
+
+
 \subsection[\pkg{varioref}:\ How to make \cs{reftex...}\ empty]
            {\pkg{varioref}:\ How to make \cs{reftexfaceafter}, etc.\ empty}
 
+%% This does not seem to make much sense! FMi: better?
```

Comment:
Yes, better (somewhat :-)
But why is it named "...faceafter" ? 
Should it be "...spaceafter" ? 

Oh, I See now: 
"...textface..."

Reply:
ah, that is better abbreviated `\reftext...`

Reply:
Still has the incorrect

Reply:
maybe you need to do some refresh of the page. It is corrected and checked in:

https://github.com/latex3/latex2e/pull/1740/files